cmd/kwild/config/flags.go

```diff
@@ -119,4 +119,6 @@ to instead run a dedicated seeder like https://github.com/kwilteam/cometseed.`))
 	flagSet.MarkDeprecated("app.snapshots.snapshot-dir", "this value is no longer configurable")
 	flagSet.String("chain.statesync.snapshot-dir", "", "Chain state sync snapshot directory")
 	flagSet.MarkDeprecated("chain.statesync.snapshot-dir", "this value is no longer configurable")
+	flagSet.BoolVar(&cfg.AppConfig.Snapshots.DEPRECATED_Enabled, "app.snapshots.enabled", cfg.AppConfig.Snapshots.DEPRECATED_Enabled, "Enable snapshots")
```

Comment:
For config.toml, we need something like this: https://github.com/kwilteam/kwil-db/pull/1013/files#diff-f23562fd52a0d43d17897596ae698df69cbb537eebd01410825af78beaa19759R154